### ISSUE TITLE
invalid link of pretrained model

### ISSUE THREAD

**Issue by kaishijeng** [rwang97/lane-and-road-detection#1]
@rwang97 

A link below of pretrained model is invalid.
>>A pre-trained model is provided in google drive. 
Please provide a valid link

Thanks

**Comment by rwang97** [comment]
Yeah sorry I actually haven't completed the README, but I will upload it tonight.

**Comment by kaishijeng** [comment]
Waiting for your new link.

I have questions about training. What BDD dataset do I need for training?
I have downloaded bdd100k_images.zip and bdd100k_labels_release.zip. Are these 2 sufficient for training?
If yes, where should I specify dataset location in your code? I saw dataset locations are hardcoded in  training.py and  utils/data_utils.py. Not sure how to change.

Thanks,

**Comment by rwang97** [comment]
Hi, I've already uploaded the link to download the model.

1. 
For BDD dataset, you'll need to download the videos and extract the frames from those videos. There are 70 zip files for the training videos, the naming is something like "bdd100k_videos_train_00.zip". 

bdd100k_images.zip only gives the labelled frames, which is not what I wanted. I needed not only the labelled frame but also several frames before that, which is the whole point of using convLSTM. As I remembered, BDD dataset only labels the frame at 10s in the video (video is ~30s long). Therefore, you'll need to extract the frames you need. For me, I extracted each frame with 0.5 second apart until the 10s.

2. 
For dataset location, you'll only need to change it in train.py. You can ignore the paths in data_utils.py since it's just for me to test whether my data loader works correctly.
